app/views/submissions/_submissions_table.js.jsx.erb

```diff
@@ -157,6 +157,11 @@
     render: function() {
       // Do student-specific table stuff here.
       // possibly move this to getdefaultprops.
+      num_completed_assignments = this.state.submissions.filter(function(submission) {
+                return (submission['marking_state'].indexOf("Completed") > -1);
```

Comment:
@hootiehoo You should do the same thing you're doing for the Admin dashboard page.